fbb/wl2k_test.go

```diff
@@ -187,3 +187,44 @@ func TestSessionCMSv4(t *testing.T) {
 		t.Errorf("Session exchange returned error: %s", err)
 	}
 }
+
+func TestSortProposals(t *testing.T) {
+	props := make([]*Proposal, 5)
+	props[0], _ = proposalWithSubject("Just a test")
+	props[1], _ = proposalWithSubject("Re://WL2K O/Very important")
+	props[2], _ = proposalWithSubject("//WL2K R/Read this sometime, or don't")
+	props[3], _ = proposalWithSubject("//WL2K P/ Pretty important")
+	props[4], _ = proposalWithSubject("//WL2K Z/The world is on fire!")
```

Comment:
An alternative way to declare this would be:

```go
props := []*Proposal{
  mustProposalWithSubject("Just a test"),
  mustProposalWithSubject("Re://WL2K O/Very important"),
  // ...
}
```

... the [must-pattern](https://www.journal.petrausch.info/post/2020/05/must-pattern/) is rather common in Go, especially in tests like these where it is not that big of a deal if the code panics. It's even used in the standard library in some cases (like regexp.MustCompile).

```go
func mustProposalWithSubject(subject string) *Proposal {
  p, err := proposalWithSubject(subject)
  if err != nil {
    panic(err)
  }
  return p
}
```

Reply:
Incorporated must-pattern